src/components/Dropdown/Dropdown.js

```diff
@@ -27,18 +27,18 @@ export default function Dropdown({ options, prompt, value, key, onChange, }) {
         <div className={`options ${open ? "open" : null}`}>
           {options.map((option) => (
                 <div
-                    className={`option ${value === option ? "selected" : null}`}
-                    key = {option}
+                  className={`option ${value === option ? "selected" : null}`}
+                  key = {option}
                   onClick={() => {
-                  onChange(option);
-                  setOpen(false);
-                  key = "{option}"
-                }}>
-                  {option}</div>
+                    onChange(option);
+                    setOpen(false);
+                    key = "{option}"
```

Comment:
I left an extra key in here, we don't need line 35. I'll remove it before my next PR